lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -1,6 +1,4 @@
-define( (require, exports, module) ->
-  PapeeriaLatexHighlightRules = require('./papeeria_latex_highlight_rules')
-  LatexParsingContext = require('./latex_parsing_context')
+define(["./papeeria_latex_highlight_rules", './latex_parsing_context'],  (PapeeriaLatexHighlightRules, LatexParsingContext) ->
```

Comment:
We normally use double-quotes in string literals, and at least use the same style throughout a single file